addons/l10n_es_edi_facturae/demo/res_company_demo.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_es.partner_demo_company_es" model="res.partner">
+        <field name="zip">12345</field>
```

Comment:
It's weird to change the zip for this ES partner when installing the EDI. Change the partner in l10n_es directly?